### DISCUSSION TITLE
Add a configuration for setting custom packageDiff url ( https://renovatebot.com/diffs/npm/ )

### DISCUSSION THREAD

**Discussion by sebastienroux** [renovatebot/renovate#32082] [Request Help]
Hello,

I have a self hosted gitlab and renovate bot, i wrote a custom package diff tool for my private npm repository.

I would like to be able to change the packageDiff url https://renovatebot.com/diffs/npm/ for my custom url.

Could you add this ? 

Thanks :)

**Comment by rarkins** [comment]
You can change this yourself by modifying `prBodyDefinitions` for npm
